Misc/NEWS.d/next/Security/2019-05-21-23-20-18.bpo-35907.NC_zNK.rst

```diff
@@ -0,0 +1,2 @@
+CVE-2019-9948: Avoid file reading as disallowing the unnecessary URL scheme in
```

Comment:
This text doesn't make sense as written.  "as disallowing the unnecessary URL scheme"  says nothing.  _WHAT_ url scheme did we just break our users applications by refusing to support?  Those need to be listed in the NEWS entry.

```
CVE-2019-9948: Drop support for the xxx_yyy URL scheme in ...
```